src/core_modules/capture-core/components/WidgetEventEdit/ViewEventDataEntry/viewEventDataEntry.actions.js

```diff
@@ -42,20 +42,23 @@ export const loadViewEventDataEntry =
         program,
         enrollment,
         attributeValues,
+        dataEntryId,
+        dataEntryKey,
     }: {
         eventContainer: ClientEventContainer,
         orgUnit: OrgUnit,
         foundation: RenderFoundation,
         program: Program,
+        dataEntryId: string,
+        dataEntryKey: string,
         enrollment?: EnrollmentData,
         attributeValues?: Array<AttributeValue>,
     }) => {
-        const dataEntryId = viewEventIds.dataEntryId;
-        const itemId = viewEventIds.itemId;
         const dataEntryPropsToInclude = [
             {
                 id: 'occurredAt',
                 type: 'DATE',
+                validatorContainers: getEventDateValidatorContainers(),
```

Comment:
The main point in this fix? No idea how it works, but I suppose it does :)

Reply:
Hey @superskip, 

This line adds validation checks to the Report Date field. Now if the date doesn't have a valid value, an error message will appear below the field and the user will not be able to save the event until the error is fixed.  

Thanks for the review!